docs/client/frontend.md

```diff
@@ -45,6 +45,24 @@ console.log(movies)
 
 You can use both named operations and the factory in the same file. They can work on different hosts, so the factory does _not_ use the global `setBaseUrl` function.
 
+### Default Headers
```

Comment:
please document also the "global" `setDefaultHeaders()`

Reply:
done!